### PR TITLE
clarification of variation in R0 estimates

### PR DESCRIPTION
**Pull request by rdvelazquez**
### Description of the proposed additions or changes

Adds some clarification on how to interpret the variability in estimated values of R0. @alavendelm as you wrote the initial draft... let me know if you think this is (1) correct and (2) a worthwhile clarification. If you think the variability in estimated R0 came more from the methods used than from the different populations then we could discard this suggested clarification or modify it in some way.

### Related issues

#313 

### Suggested reviewers (optional)

@alavendelm @yemarshall 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

### CODE REVIEW

File: content/07.pathogenesis.md [hunk 1]
```diff
@@ -55,7 +55,8 @@ When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the popu
 R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
 Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74].
 
-R~0~ for COVID-19 is estimated to lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221]; estimates vary considerably depending on the data and the methods used.
+Estimates of R~0~ for COVID-19 lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221].
+Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations at specific times, therefore the different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.
```
Comment (rando2) [inline content/07.pathogenesis.md:59]:
```suggestion
Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations at specific times; therefore, the different estimates of R~0~ should not be interpreted as a range of estimates of the same underlying parameter.
```

**Comment by rando2** [comment]
I am happy to merge but thought it might be good to ask @alavendelm to approve first!

**Comment by alavendelm** [comment]
oh!! I thought I commented on this few days ago but I started a review and was left "pending"
now deleted 🤦 

I have a concern about the last clause - don't think it is necessarily true, there is, in theory, one R0 for coronavirus in a human population. Will suggest alternative

File: content/07.pathogenesis.md (hunk 1 above)
Comment (alavendelm) [reply content/07.pathogenesis.md:59]:
```suggestion
Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations in specific environments. The different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.
```

Comment (alavendelm) [reply content/07.pathogenesis.md:59]: ok just changed ref to time since R0 should - by definition - be time-independent

**Comment by rdvelazquez** [comment]
Thanks for the review @alavendelm (that "pending" review feature gets all of us occasionally).

After I accepted your first edit, GitHub marked it as "resolved" and wouldn't let me also accept the second edit so I made that change as a separate commit (https://github.com/greenelab/covid19-review/pull/352/commits/4932687a27d5691cf6d1e664f404e274862f08f0).

**Pull request merged**